src/table/context/TableContext.tsx

```diff
@@ -42,18 +42,18 @@ export const TableContextProvider = ({
   viewService,
   isNewHeadCellMenuEnabled,
 }: ContextProviderProps) => {
+  const featureFlags = { isNewHeadCellMenuEnabled };
   const [headRowHeight, setHeadRowHeight] = useState(0);
   const [focusedCellCoord, setFocusedCellCoord] = useState<[number, number]>(cellCoordMock || FIRST_HEADER_CELL_COORD);
   const [selectionState, selectionDispatch] = useSelectionReducer(tableData.rows, selectionsAPI);
   const [hoverIndex, setHoverIndex] = useState(-1);
-  const styling = useTableStyling(layout, theme, tableData, rootElement);
+  const styling = useTableStyling(layout, theme, tableData, rootElement, featureFlags);
   const [columnWidths, setColumnWidths, setYScrollbarWidth, showRightBorder] = useColumnWidths(
     tableData.columns,
     tableData.totalsPosition,
     rect.width,
-    styling,
+    styling
```

Comment:
why the remove of trailing commas? lint rule change?

Reply:
I think so, I just pilled the latest main and branched off based on that, my take is that probably when i did a yarn install -> it picked up some new versions of qlik based lint settings and those were updated